scripts/samplesheet_generator.py

```diff
@@ -294,6 +294,56 @@ def gen_Miseq_data(pro):
     return ("{}{}".format(header, str_data), data)
 
 
+def gen_Nextseq_lane_data(pro):
+    data=[]
+    header_ar = ["FCID","Lane","Sample_ID","Sample_Name","Sample_Ref","index","index2","Description","Control","Recipe","Operator","Sample_Project"]
+    for out in pro.all_outputs():
+        if  out.type != "Analyte":
+            continue
+        for sample in out.samples:
```

Comment:
A detail, but this could be simplified to:

```python
if out.type == "Analyte":
    for sample in out.samples:
    ...
```

Reply:
fixed